components/google-cloud/google_cloud_pipeline_components/experimental/__init__.py

```diff
@@ -39,15 +40,17 @@ def ForecastingPreprocessingOp(**kwargs):
       this arg. The value will be automatically overriden by Managed Pipeline.
 
   Args:
-    **kwargs: Arbitrary keyword arguments.
+    **kwargs: See Accepted kwargs.
```

Comment:
Not blocking, this can be handled in a follow up PR. I was just wondering why using kwargs here instead of using a named list of args.  **kwargs is a good choice for backwards compatibility and if list of args are not known, however in this case we do know the list downstream parameters might be a good choice to directly call them out here.